tests/visibility_negative/visibility_negative.fz

```diff
@@ -126,118 +126,118 @@ visibility_negative is
       use f8   // 39. should flag an error: feature not found / not visible
       use f9   // 40. should flag an error: feature not found / not visible
       b1 is
-        chck (f1 == 11 || f1 == 77 || f1 == 143) "can access f1"
+        chck (f1 = 11 || f1 = 77 || f1 = 143) "can access f1"
         use f2   // 41. should flag an error: feature not found / not visible
         use f3   // 42. should flag an error: feature not found / not visible
         use f4   // 43. should flag an error: feature not found / not visible
         use f5   // 44. should flag an error: feature not found / not visible
         use f6   // 45. should flag an error: feature not found / not visible
         use f7   // 46. should flag an error: feature not found / not visible
-        use f8   // 47. should flag an error: feature not found / not visible
+        use f8   // 47. NYI: UNDER DEVELOPMENT: flag an error: feature not found / not visible
```

Comment:
I think i do not agree with these test. `f8` is defined in `a1` and used here in `b1`. Should IMHO be fine even though it defined "later" in the source text than `b1`. I think the DFA needs to find if this can lead to uninitialized access?